include/course.h

```diff
@@ -127,9 +127,9 @@ extern f32 gCourseNearPersp;
 #define gCourseFarPersp
 #define gCourseNearPersp
 #define GET_COURSE_D_0D0096B8(cc)
-#define GET_COURSE_D_0D009808(cc)
-#define GET_COURSE_D_0D009418(cc)
-#define GET_COURSE_D_0D009568(cc)
+#define GET_COURSE_gCpuOffTrackTargetSpeed(cc)
+#define GET_COURSE_gCpuCurveTargetSpeed(cc)
+#define GET_COURSE_gCpuNormalTargetSpeed(cc)
```

Comment:
CPU

Reply:
Didn't we decide a while ago weather we should use CPU or AI or other?
I think we decided on AI.

I don't mind CPU. it just needs to be consistent

Reply:
I remember the invert